Enable next extensions in the sundown example:
  * tables
  * fenced_code
  * autolink
  * strikethrough
  * space_headers
  * superscript
See `markdown.h` to calculate mkd_extensions value.

--- a/examples/sundown.c
+++ b/examples/sundown.c
@@ -62,7 +62,7 @@
 	ob = bufnew(OUTPUT_UNIT);
 
 	sdhtml_renderer(&callbacks, &options, 0);
-	markdown = sd_markdown_new(0, 16, &callbacks, &options);
+	markdown = sd_markdown_new(222, 16, &callbacks, &options);
 
 	sd_markdown_render(ob, ib->data, ib->size, markdown);
 	sd_markdown_free(markdown);
